.circleci/config.yml

```
@@ -1254,6 +1254,17 @@ jobs:
           event: fail
           template: basic_fail_1
 
+  op-program-compat:
+    docker:
+      - image: cimg/go:1.21
+    steps:
+      - checkout
+      - run:
+          name: compat-goerli
+          command: |
+            make run-goerli-verify
+          working_directory: op-program
```

```suggestion
  op-program-compat:
    docker:
      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest
    steps:
      - checkout
      - restore_cache:
          name: Restore Go modules cache
          key: gomod-{{ checksum "go.sum" }}
      - restore_cache:
          key: golang-build-cache
      - run:
          name: compat-goerli
          command: |
            make run-goerli-verify
          working_directory: op-program
```

To reuse the Go build/mod caches.